packages/contracts/deploy/30_plugins/10_plugin-repos/00_create-plugins-repo.ts

```diff
@@ -18,53 +22,87 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const {network} = hre;
 
   // AddresslistVotingSetup
-  const addresslistCIDPath = await uploadToIPFS(
-    JSON.stringify(addresslistMetadata),
+  const addresslistReleaseCIDPath = await uploadToIPFS(
+    JSON.stringify(addresslistReleaseMetadata),
+    network.name
+  );
+  const addresslistBuildCIDPath = await uploadToIPFS(
+    JSON.stringify(addresslistBuildMetadata),
     network.name
   );
   await createPluginRepo(
     hre,
     'AddresslistVoting',
     'AddresslistVotingSetup',
     ethers.utils.hexlify(
-      ethers.utils.toUtf8Bytes(`ipfs://${addresslistCIDPath}`)
+      ethers.utils.toUtf8Bytes(`ipfs://${addresslistReleaseCIDPath}`)
+    ),
+    ethers.utils.hexlify(
+      ethers.utils.toUtf8Bytes(`ipfs://${addresslistBuildCIDPath}`)
     )
   );
 
   // TokenVotingSetup
-  const tokenCIDPath = await uploadToIPFS(
-    JSON.stringify(tokenMetadata),
+  const tokenReleaseCIDPath = await uploadToIPFS(
+    JSON.stringify(adminReleaseMetadata),
+    network.name
+  );
+  const tokenBuildCIDPath = await uploadToIPFS(
+    JSON.stringify(adminBuildMetadata),
     network.name
   );
   await createPluginRepo(
     hre,
     'TokenVoting',
     'TokenVotingSetup',
-    ethers.utils.hexlify(ethers.utils.toUtf8Bytes(`ipfs://${tokenCIDPath}`))
+    ethers.utils.hexlify(
+      ethers.utils.toUtf8Bytes(`ipfs://${tokenReleaseCIDPath}`)
+    ),
+    ethers.utils.hexlify(
+      ethers.utils.toUtf8Bytes(`ipfs://${tokenBuildCIDPath}`)
+    )
   );
 
   // AdminSetup
-  const adminCIDPath = await uploadToIPFS(
-    JSON.stringify(adminMetadata),
+  const adminReleaseCIDPath = await uploadToIPFS(
+    JSON.stringify(multisigReleaseMetadata),
```

Comment:
wrong metadata and check below as well (not correct for multisig as well).

Reply:
amended